### ISSUE TITLE
[beta] ix-f importer failure

### ISSUE THREAD

**Issue by vegu** [peeringdb/peeringdb#1355]
**Describe the bug**

beta.peeringdb.com is currently experiencing an ix-f import failure for some exchanges. cause appears to be a bug with the changes for #1271 that were rolled out to beta this week.

```
File "/srv/www.peeringdb.com/peeringdb_server/management/commands/pdb_ixf_ixp_member_import.py", line 278, in handle
    success = importer.update(
  File "/srv/www.peeringdb.com/peeringdb_server/ixf.py", line 582, in update
    self.cleanup_aged_proposals()
  File "/srv/www.peeringdb.com/venv/lib/python3.9/site-packages/reversion/revisions.py", line 324, in do_revision_context
    return func(*args, **kwargs)
  File "/srv/[www.peeringdb.com/peeringdb_server/ixf.py](http://www.peeringdb.com/peeringdb_server/ixf.py)", line 1898, in cleanup_aged_proposals
    ixf_member_data.netixlan.delete()
  File "/srv/www.peeringdb.com/venv/lib/python3.9/site-packages/django_handleref/models.py", line 137, in delete
    self.save()
  File "/srv/www.peeringdb.com/peeringdb_server/models.py", line 5456, in save
    super().save(*args, **kwargs)
  File "/srv/www.peeringdb.com/venv/lib/python3.9/site-packages/django/db/models/base.py", line 739, in save
    self.save_base(using=using, force_insert=force_insert,
  File "/srv/www.peeringdb.com/venv/lib/python3.9/site-packages/django/db/models/base.py", line 776, in save_base
    updated = self._save_table(
  File "/srv/[www.peeringdb.com/venv/lib/python3.9/site-packages/django/db/models/base.py](http://www.peeringdb.com/venv/lib/python3.9/site-packages/django/db/models/base.py)", line 881, in _save_table
    results = self._do_insert(cls._base_manager, using, fields, returning_fields, raw)
  File "/srv/[www.peeringdb.com/venv/lib/python3.9/site-packages/django/db/models/base.py](http://www.peeringdb.com/venv/lib/python3.9/site-packages/django/db/models/base.py)", line 919, in _do_insert
    return manager._insert(
  File "/srv/www.peeringdb.com/venv/lib/python3.9/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/srv/www.peeringdb.com/venv/lib/python3.9/site-packages/django/db/models/query.py", line 1270, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "/srv/www.peeringdb.com/venv/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1416, in execute_sql
    cursor.execute(sql, params)
```

**What is the proposed priority?**
needs to be fixed before version 2.45 can be rolled to production

**Comment by ccaputo** [comment]
> **What is the proposed priority?** needs to be fixed before version 2.45 can be rolled to production

Agreed. We can't have a broken IXF-Importer on prod.

**Comment by grizz** [comment]
Moving to 4, clear bug.
